go.sum

```diff
@@ -26,8 +26,10 @@ github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZN
 github.com/stretchr/objx v0.1.0/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
 github.com/stretchr/testify v1.6.1 h1:hDPOHmpOpP40lSULcqw7IrRb/u7w6RpDC9399XyoNd0=
 github.com/stretchr/testify v1.6.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
-github.com/ulikunitz/xz v0.5.8 h1:ERv8V6GKqVi23rgu5cj9pVfVzJbOqAY2Ntl88O6c2nQ=
-github.com/ulikunitz/xz v0.5.8/go.mod h1:nbz6k7qbPmH4IRqmfOplQw/tblSgqTqBwxkY0oWt/14=
+github.com/ulikunitz/xz v0.5.6/go.mod h1:2bypXElzHzzJZwzH67Y6wb67pO62Rzfn7BSiF4ABRW8=
+github.com/ulikunitz/xz v0.5.7/go.mod h1:nbz6k7qbPmH4IRqmfOplQw/tblSgqTqBwxkY0oWt/14=
```

Comment:
Hmmm I think the way these vulnerability scanners work is that they scan the go.sum for references to the package versions with the CVE present. I'm pretty sure this is going to make it break again if there is still reference to github.com/ulikunitz/xz v0.5.6.

Can you investigate a little and find out?

Thanks @mishas

Reply:
The way Go works, is it's taking the largest version (that's still considered to be backward compatible) from the go.sum file.
If the vulnerability scanner is any good, it should do the same...

I can only speak towards Dependabot (the Github built-in solution), which is what I'm using, and it will works correctly.
No idea what others are using, and how that will stack up to the challenge.

Reply:
Hey @mishas,

It's been something I've been meaning to do for a while anyway so have started on the task this evening of removing the actual majority of the archiver dependencies, which would include xz anyway.

If you're desperate for the above change we can work this one through but I think I should be able to get a release of the removal in the next few days out for you.

If you'd like to contribute to the code or review you can find here https://github.com/fergusstrange/embedded-postgres/pull/50. It's almost there but will need some eyes on it and some new coverage.

Thanks